bundles/org.openhab.binding.anthem/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="anthem"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="anthem">
+		<label>anthem</label>
```

Comment:
```suggestion
		<label>Anthem</label>
```

Reply:
Fixed

Reply:
And now the I18N file needs to be regenerated - the price of redundancy. 😉

Reply:
Ugh. I can almost guarantee I'll forget to do that again in the future. 😭

Reply:
And I had to hand-edit the file after running the tool. Since I changed the name of a config parameter, I had to delete the old properties it left in the file. 🙄